pandas/plotting/_core.py

```diff
@@ -720,16 +720,24 @@ class PlotAccessor(PandasObject):
 
            Now applicable to planar plots (`scatter`, `hexbin`).
 
+        .. versionchanged:: 1.6.0
+
+            Now applicable to histograms.
+
     ylabel : label, optional
         Name to use for the ylabel on y-axis. Default will show no ylabel, or the
         y-column name for planar plots.
 
         .. versionadded:: 1.1.0
 
-        .. versionchanged:: 1.2.0
+        .. versionchanged:: 2.0.0
```

Comment:
I think this should remain 1.2 since it applies to the scatter/hexbin note below

Reply:
Good catch &ndash; not sure why this got changed.